code/game/mecha/mecha.dm

```diff
@@ -263,7 +264,7 @@
 		return 1
 
 /obj/mecha/relaymove(mob/user, direction)
-	if(!direction)
+	if(!direction || frozen)

```

Comment:
perhaps better to check if the occupant is frozen? 
if(occupant in frozen_mob_list)
    he_froze_yo

Reply:
Well see I also want mechs themselves able to be frozen, as I think that's the intent of the freeze mech verb

Reply:
Boom you got it :P 
But I can see the need for freezing mechs. For instance if the AI is using it